### DISCUSSION TITLE
Agent in big trouble: NodeId=qtMciSO87a4FUwAA1wNAqhEoSLagDv3DUOWexBhBmDsuFzkxfYNM9NA1m7jsHNJ1, IP=x.x.x.x:2677, Agent=25.

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#5712] [Issues] [answered]
Hi,

this is shown when running MeshCentral from Console with --debug main.

`Agent in big trouble: NodeId=qtMciSO87a4FUwAA1wNAqhEoSLagDv3DUOWexBhBmDsuFzkxfYNM9NA1m7jsHNJ1, IP=x.x.x.x:2677, Agent=25.`

What does this mean? How do fix? Or does it need to be fixed?

Regards

**Comment by si458** [comment, marked as answer]
plz dont double post, stick to either an issue or a discussion!? #5711

EDIT: code shows this message before that event happens, so it seems that agent/device must be disconnecting and reconnecting too often, so the is a connection issue with the remote device
```
// Check how many times this agent disconnected in the last few minutes.
const disconnectCount = parent.wsagentsDisconnections[obj.nodeid];
if (disconnectCount > 6) {
  parent.parent.debug('agent', 'Agent in big trouble: NodeId=' + obj.nodeid + ', IP=' + obj.remoteaddrport + ', Agent=' + obj.agentInfo.agentId + '.');
```

**Reply by JSuenram** [reply]
Is not a double post.... another server... another customer... another issue.....
But "to much reconnects" is possible and ok for me as answer....

Thank you!
